course/layouts/partials/video.html

```diff
@@ -28,7 +31,7 @@
 
   {{if $captionsLocation}}
```

Comment:
In this and other places where you're calling template logic, the standard I've been following is to leave 1 space on either side of the curly braces.  For example, `{{ if $captionsLocation }}`.  I'm not sure how necessary this is, but in every sample of Hugo template code I've ever seen this pattern is always followed.  Again I'm not sure exactly why or if it even matters, but I follow this pattern because it's what I see in every example I've ever looked at.